examples/mouse_drag_callback.py

```diff
@@ -2,8 +2,8 @@
 Mouse drag callback
 ===================
 
-Example updating the status bar with line profile info while dragging
-lines around in a shapes layer.
+Example printing line profile averages to the terminal while
+dragging lines in a shapes layer over an image.
```

Comment:
```suggestion
This example shows how to use a mouse drag callback in napari to compute
and print the average intensity along a line Shape as it is moved over an Image layer.
```
I just saw your comment on the issue -- sorry! -- I like the extra detail you had in the 2nd option there.

Reply:
I like this suggestion as well (or some variant that includes the more descriptive version from https://github.com/napari/napari/issues/8016#issuecomment-2969954903)

Thank you!